external-crates/move/crates/move-compiler/src/linters/mod.rs

```diff
@@ -15,6 +15,17 @@ pub enum LintLevel {
     All,
 }
 
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+#[repr(u8)]
+pub enum LinterDiagCategory {
+    Correctness,
+    Complexity,
+    Suspicious,
+    Deprecated,
+    Style,
+    Sui = 99,
```

Comment:
Can we come up with a better number than `99`? Maybe`51` (short for sui) or u8::MAX?

Reply:
Its limited to two digits, so it has to be <= 99. I took 99 since it sort of starting from the max and working backwards.